src/graphblas/nonblocking/init.cpp

```diff
@@ -38,7 +38,7 @@ bool grb::internal::NONBLOCKING::warn_if_not_native = true;
 bool grb::internal::NONBLOCKING::manual_tile_size = false;
 size_t grb::internal::NONBLOCKING::manual_fixed_tile_size =
 	grb::config::ANALYTIC_MODEL::MIN_TILE_SIZE;
-size_t grb::internal::NONBLOCKING::num_threads = grb::config::OMP::threads();
+size_t grb::internal::NONBLOCKING::num_threads; // = grb::config::OMP::threads();
```

Comment:
What's happening here btw, was this a circular dependence? (If yes, that's a straight up bug right??)